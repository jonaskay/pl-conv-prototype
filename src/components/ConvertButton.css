.ConvertButton {
  background: #536DFE;
  border: 1px solid #536DFE;
  border-radius: .25rem;
  box-shadow: 
    0 3px 5px -1px rgba(0,0,0,.2), 0 6px 10px 0 rgba(0,0,0,.14), 0 1px 18px 0 rgba(0,0,0,.12);
  color: #FFF;
  cursor: pointer;
  font-size: 1.25rem;
  font-weight: 700;
  padding: .5rem .75rem;
  width: 100%;
}
