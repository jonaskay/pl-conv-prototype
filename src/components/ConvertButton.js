import React from 'react';
import './ConvertButton.css';

class ConvertButton extends React.Component {
  render() {
    return (
      <button className="ConvertButton">Convert Playlist</button>
    );
  }
}

export default ConvertButton;
