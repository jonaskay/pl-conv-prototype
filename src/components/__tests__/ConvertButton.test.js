import React from 'react';
import ReactTestRenderer from 'react-test-renderer';
import ConvertButton from '../ConvertButton';

it('renders a button', () => {
  const component = ReactTestRenderer.create(<ConvertButton />);
  expect(component).toMatchSnapshot();
});
