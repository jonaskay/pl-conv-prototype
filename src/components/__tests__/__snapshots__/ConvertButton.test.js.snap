// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a button 1`] = `
<button
  className="ConvertButton"
>
  Convert Playlist
</button>
`;
